packages/nx/src/generators/tree.spec.ts

```diff
@@ -331,59 +334,72 @@ describe('tree', () => {
         expect(tree.children('parent/new-child')).toEqual([
           'new-child-file.txt',
         ]);
+        expect(tree.children('parent', true)).toEqual([
+          'child/child-file.txt',
+          'child/child-file2.txt',
+          'parent-file-with-write-options.txt',
+          'parent-file.txt',
+          'new-child/new-child-file.txt',
+        ]);
       });
-
       it('should return the list of children after renaming', () => {
         tree.rename(
           'parent/child/child-file.txt',
           'parent/child/renamed-child-file.txt'
         );
-
         expect(tree.children('parent/child')).toEqual([
           'renamed-child-file.txt',
         ]);
-
         tree.rename(
           'parent/child/renamed-child-file.txt',
           'parent/renamed-child/renamed-child-file.txt'
         );
-
         expect(tree.children('parent')).toEqual([
           'parent-file-with-write-options.txt',
           'parent-file.txt',
           'renamed-child',
         ]);
+        expect(tree.children('', true)).toEqual([
```

Comment:
Tree can't track when renaming files or directories. Do you know how to solve this ?